Entities/Common/GUI/ActorHUDStartPos.as

```diff
@@ -31,10 +31,13 @@ void DrawInventoryOnHUD(CBlob@ this, Vec2f tl)
 		const string name = item.getName();
 		if (drawn.find(name) == -1)
 		{
-			const int quantity = this.getBlobCount(name);
+			const int quantity = inv.getCount(name);
 			drawn.push_back(name);
 
-			GUI::DrawIcon(item.inventoryIconName, item.inventoryIconFrame, item.inventoryFrameDimension, tl + Vec2f(0 + (drawn.length - 1) * 40, -6), 1.0f);
+			Vec2f iconpos = tl + Vec2f((drawn.length - 1) * 40, -6);
+			iconpos.x += Maths::Clamp(16 - item.inventoryFrameDimension.x, -item.inventoryFrameDimension.x, item.inventoryFrameDimension.x);
+			iconpos.y += Maths::Max(16 - item.inventoryFrameDimension.y, 0);
+			GUI::DrawIcon(item.inventoryIconName, item.inventoryIconFrame, item.inventoryFrameDimension, iconpos, 1.0f, item.getTeamNum());
```

Comment:
See comment, unconditionally applying the team color is undesirable due to certain sprites using the KAG blue tints as actual colors

Reply:
Seems more like a sprite issue than a code issue.
There is currently three sprites where the tint is a concern: bucket.png, killfeedicons.png, and emoticons.png.
Ive edited bucket.png and killfeedicons.png so the water parts don't change color. Emoticons.png has its own fix in #1047 so I left it out.